src/LTDBeget/MessagePackRpc/Back.php

```diff
@@ -256,8 +256,7 @@ public function serverSendObject($code, $sets, $errs)
     public function serverRecvObject($recv)
     {
         $data = $this->serializer->unserialize($recv);
-
-        if (count($data) != 4) {
+        if (count($data) != 5) {
```

Comment:
> is 5 correct?

yeah. Because a [new field](https://github.com/LTD-Beget/msgpackrpc/commit/4e429585df0b381fe1fd9b6a179e90b285385696#diff-67a2861f9556723b99a787aa8775ddbbdafcf257811db03c497fb98d78cf54d7R88) was added earlier, which broke this check.  
I've fixed it now